### PR TITLE
Fix/survey thanks page

### PR DESCRIPTION
**Pull request by PedroErnst**
Make the survey redirect to an entry point instead of pointing to the php file directly to avoid htaccess issues.
Pass the survey name as GET parameter to be able to display it.

## Motivation and Context

Issue #4789 

## How To Test This

Create a survey.
Answer the question.
Verify the thanks page opens and displays the correct survey name.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Surveys/Entry/Thanks.php [hunk 1]
```diff
@@ -14,14 +20,9 @@
...
 <div class="container">
-    <div class="row">
-        <div class="col-md-offset-3 col-md-6">
-            <img height=100 src="survey_logo.jpg"/>
-        </div>
-    </div>
     <div class="row well">
         <div class="col-md-offset-2 col-md-8">
-            <h1><?= $survey->name ?></h1>
+            <h1><?php echo $surveyName; ?></h1>
```
Comment (daniel-samson) [inline modules/Surveys/Entry/Thanks.php]: Question: The short hand tags "<?=" are available on php 5.4 and above. SuiteCRM 7.10 has a minimum requirement of 5.5.9. So why do we need to change these?

Comment (PedroErnst) [reply modules/Surveys/Entry/Thanks.php]: PSR-1

Comment (PedroErnst) [reply modules/Surveys/Entry/Thanks.php]: Yes my bad, that change is unnecessary

Comment (daniel-samson) [reply modules/Surveys/Entry/Thanks.php]: Ah OK. I wanted to make sure that the community member who raised it, was using the correct php version.

**Pull request merged**